redbot/cogs/audio/core/events/cog.py

```diff
@@ -25,6 +25,10 @@ async def on_red_audio_track_start(
     ):
         if not (track and guild):
             return
+
+        if not await self.bot.cog_disabled_in_guild(self, guild):
```

Comment:
Something I just noticed, should `not` be here?
I'm still leaving Audio to people mentioned earlier (unless they won't have time to look at it), but this seems wrong here.